source/java/src/main/java/com/uber/neuropod/NeuropodValue.java

```diff
@@ -0,0 +1,365 @@
+package com.uber.neuropod;
+
+import java.io.Serializable;
+import java.nio.*;
+import java.util.List;
+
+/**
+ * This class is an one to one mapping to cpp NeuropodValue type.
+ * Should not call the close function if this class does not have the ownership of the underlying native class.
```

Comment:
These comments are outdated, they are only valid before I add reference counter and reference manager to the code. I'll delete them. Now we can call the close function in whatever case.
A class has the ownership of an underlying means it is responsible for the C++ side memory. A Java side copy may make a C++ object owned by two Java object and this will cause a problem without the reference manager in the C++ side.